test/gfx/test.sh

```diff
@@ -42,14 +42,23 @@ rm -f out*.png result.png
 
 for f in *.png; do
 	flags="$([[ -e "${f%.png}.flags" ]] && echo "@${f%.png}.flags")"
-	new_test ../../rgbgfx $flags "$f"
 
+	new_test ../../rgbgfx $flags "$f"
 	if [[ -e "${f%.png}.err" ]]; then
 		test 2>"$errtmp"
 		diff -u --strip-trailing-cr "${f%.png}.err" "$errtmp" || fail
 	else
 		test || fail $?
 	fi
+
+	new_test ../../rgbgfx $flags - "<$f"
```

Comment:
Now with more than one usage, this could use `RGBGFX=../../rgbgfx` like the other test scripts do.